src/test/java/seedu/address/commons/core/ConfigTest.java

```diff
@@ -14,7 +14,7 @@
 
     @Test
     public void toString_defaultObject_stringReturned() {
-        String defaultConfigAsString = "App title : Address App\n"
+        String defaultConfigAsString = "App title : Death Note\n"
```

Comment:
Hopefully there are no more instances of Address App lying around. Take note guys.